Tools/Create-Azure-Sentinel-Solution/common/templating/contentPackages.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
What is the use of this file? Can't we create this while generating the CCP resources itself?

Reply:
removed